src/main/java/com/fitwsarah/fitwsarah/appointmentsubdomain/presentationlayer/AppointmentRequestModel.java

```diff
@@ -11,16 +11,16 @@
 @AllArgsConstructor(access = AccessLevel.PUBLIC)
 public class AppointmentRequestModel {
 
-    private String availabilityId;
-    private String userId;
-    private String serviceId;
-    private Status status;
-    private String location;
-    private String firstName;
-    private String lastName;
-    private String phoneNum;
-    private String date;
-    private String time;
+    String availabilityId;
+    String userId;
```

Comment:
Is a there a reason these are no longer private?

Reply:
they are already public